app/src/main/AndroidManifest.xml

```diff
@@ -8,7 +9,15 @@
         android:label="@string/app_name"
         android:theme="@style/AppTheme" >
         <activity
-            android:name=".ui.request_flow.RequestStateActivity"
+            android:name=".ui.request_flow.Activities.SelectShopperActivity"
+            android:label="@string/title_activity_select_shopper" >
+          <intent-filter>
+            <action android:name="android.intent.action.MAIN"/>
```

Comment:
Make sure you remove these before the merge.
Only the `HomeActivity` should have MAIN/LAUNCHER.